WalletWasabi/Blockchain/TransactionBuilding/BnB/LessSelectionStrategy.cs

```diff
@@ -0,0 +1,40 @@
+namespace WalletWasabi.Blockchain.TransactionBuilding.BnB;
+
+public class LessSelectionStrategy : SelectionStrategy
+{
+	public LessSelectionStrategy(long target, long[] inputValues, long[] inputCosts) : base(target, inputValues, inputCosts)
+	{
+		BestTargetSoFar = long.MinValue;
+	}
+
+	public override EvaluationResult Evaluate(long[] selection, int depth, long sum)
+	{
+		long totalCost = sum + CurrentInputCosts;
+
+		if (totalCost > Target)
```

Comment:
Isn't this supposed to be 
```suggestion
		if (sum > Target)
```
?